mandatory/2-function-creation.js

```diff
@@ -43,7 +43,7 @@ Write a function that:
 - makes all strings all lowercase
 */
 function tidyUpStrings(arrayOfStrings) {
-  return arrayOfStrings.map((e) => e.toLowerCase().trim().replace(/\//g, ""));
+  return arrayOfStrings.map((e) => e.toLowerCase().trim().replace("/", ""));
```

Comment:
Hey. Good that you are exploring Regular Expression (Regex).  Keep trying and you need it so much in everyday programming works!